content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -1,94 +1,131 @@
+   for managing more complex :abbr:`BoMs (bills of materials)`.
+#. :guilabel:`Subcontracting` outsources the production of some (or all) components or products to
+   outside manufacturers. This feature needs to be enabled in :menuselection:`Manufacturing app -->
+   Configuration --> Settings`, under the :guilabel:`Operations` section.
 
-Using the same BoM to describe Variants
----------------------------------------
+.. note::
+   A :abbr:`BoM (bill of materials)` can also be created directly from the :abbr:`BoM (bill of
+   materials)` smart button on the product template, in which case the :guilabel:`Product` field is
+   pre-filled.
+
+The most common :guilabel:`BoM Type` is :guilabel:`Manufacture this product`. Once the
+:guilabel:`BoM Type` is chosen, click :guilabel:`Add a Line` to add all the :guilabel:`Components`
+that go into the production of the final product, as well as the :guilabel:`Quantity` of each
+component. Finally, click :guilabel:`Save` to finish creating the :abbr:`BoM (bill of materials)`.
 
-As said above, you can use *BoM* for *Product Variants*. It is
-basically the same as for the standard product.
+.. tip::
+   New components can be created on-the-fly directly from the :abbr:`BoM (bill of materials)`. After
+   selecting :guilabel:`Add a line`, create the new component and select :guilabel:`Create` or
+   :guilabel:`Create and Edit`. Components can also be created by going to
+   :menuselection:`Manufacturing app --> Products --> Products`, and clicking :guilabel:`Create`.
 
-If your *BoM* is for one variant only, then specify which one in the
-*Product Variant* field. If not, specify the variant it is consumed
-for on each component line. You can add several variants for each
-component.
+Specify a BoM for a product variant
+-----------------------------------
 
-.. image:: bill_configuration/bills_of_materials_02.png
-    :align: center
+:abbr:`BoMs (bills of materials)` can also be assigned to specific *product variants*, with two
+setup options available to choose from.
 
-Adding a routing
-================
+.. image:: bill_configuration/bill-configuration-product-variants.png
+   :align: center
+   :alt: Bill of materials product variants and apply on variants options.
 
-You can add routing to your *BoM*. You will do this if you need to
-define a series of operations required by your manufacturing process. To
-use it, enable the *Work Orders* feature in the *Manufacturing* app
-settings.
+The first method is to create one :abbr:`BoM (bill of materials)` per product variant, by specifying
+the :guilabel:`Product Variant` every time a new :abbr:`BoM (bill of materials)` is created.
 
-.. image:: bill_configuration/bills_of_materials_03.png
-    :align: center
+The second method is to create **one** master :abbr:`BoM (bill of materials)` that contains all
+components, and specify which variant each component applies to in the :guilabel:`Apply on Variants`
+column.
```

Comment:
Ok so now we know there are two ways to assign BoMs to product variants, so... where's the walk-through? 😅 

Also, I think this a good spot to add a **seealso** for [this eLearning walk-through](https://www.odoo.com/slides/slide/manufacturing-with-product-variants-2805). Jose does a pretty good job showing exactly how to do this.